### PR TITLE
add share_with index

### PR DESCRIPTION
**Pull request by butonic**
screen shot 2017-08-30 at aug 30 2017 10 56am
screen shot 2017-08-30 at aug 30 2017 10 57am

### CODE REVIEW

File: apps/files_sharing/appinfo/Migrations/Version20170830112305.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ *
+ */
+
+namespace OCA\Files_Sharing\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use OCP\Migration\ISchemaMigration;
+
+/** add index to costly query */
+class Version20170804201253 implements ISchemaMigration {
```
Comment (individual-it) [inline apps/files_sharing/appinfo/Migrations/Version20170830112305.php]: I think you need to name the class `Version20170830112305`
image

**Review by individual-it — CHANGES_REQUESTED** [review]

File: apps/files_sharing/appinfo/Migrations/Version20170830112305.php (hunk 1 above)
Comment (butonic) [reply apps/files_sharing/appinfo/Migrations/Version20170830112305.php]: copy pasta error ...

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
@butonic please backport to stable10

**Comment by PVince81** [comment]
@butonic  @tomneedham please backport

**Pull request merged**